packages/storybook-test/stories/heading/heading.stories.tsx

```diff
@@ -56,128 +60,318 @@ type Story = StoryObj<typeof meta>;
+export const HeadingLevel6: Story = {
+  name: 'Heading 6 in een koppenstructuur',
   args: {
-    children: 'Knappe Koppen Van Droom naar Realiteit',
-    level: 2,
-  },
-  parameters: {
-    status: { type: [] },
+    children: 'Artikel 42',
+    level: 6,
   },
-};
-
-export const HeadingLevel3: Story = {
-  name: 'Heading 3',
-  args: {
-    children: 'Knappe Koppen Van Droom naar Realiteit',
-    level: 3,
+  decorators: (Story) => (
+    <>
+      <Heading level={1}>Civiel recht in Nederland</Heading>
+      <Heading level={2}>Burgerlijk Wetboek</Heading>
+      <Heading level={3}>Boek 1, Personen- en familierecht</Heading>
+      <Heading level={4}>Titel 5. Het huwelijk</Heading>
+      <Heading level={5}>Afdeling 1. Vereisten tot het aangaan van een huwelijk</Heading>
+      <Heading level={6}>{Story()}</Heading>
+      <Paragraph>
+        Zij die met elkander een huwelijk willen aangaan, mogen niet tegelijkertijd een geregistreerd partnerschap zijn
+        aangegaan.
+      </Paragraph>
+    </>
+  ),
+  globals: {
+    dir: 'ltr',
+    lang: 'nl',
+    title: 'Civiel Recht in Nederland (fragment uit het Burgerlijk Wetboek)',
   },
   parameters: {
+    docs: {
+      description: {
+        story: `Bron: [Artikel 42 Burgerlijk Wetboek Boek 1](https://wetten.overheid.nl/jci1.3:c:BWBR0002656&boek=1&titeldeel=5&afdeling=1&artikel=42&z=2024-01-01&g=2024-01-01)`,
+      },
+    },
     status: { type: [] },
   },
 };
 
-export const HeadingLevel4: Story = {
-  name: 'Heading 4',
+export const HeadingMultiline: Story = {
+  name: 'Heading 1 met meerdere regels tekst',
   args: {
-    children: 'Knappe Koppen Van Droom naar Realiteit',
-    level: 4,
+    children: (
+      <>
+        When the pawn hits the conflicts he thinks like a king
+        <br />
+        What he knows throws the blows when he goes to the fight
+        <br />
+        And he'll win the whole thing 'fore he enters the ring
+        <br />
+        There's no body to batter when your mind is your might
+        <br />
+        So when you go solo, you hold your own hand
+        <br />
+        And remember that depth is the greatest of heights
+        <br />
+        And if you know where you stand, then you know where to land
+        <br />
+        And if you fall it won't matter, cuz you'll know that you're right
+      </>
+    ),
+    level: 1,
+  },
+  decorators: (Story) => (
+    <>
+      {Story()}
+      <Paragraph>
+        "When the Pawn..." is the second studio album by the American singer-songwriter Fiona Apple. Released by Epic
+        Records in the United States on November 9, 1999, When the Pawn... was wholly written by Apple, with production
+        by Jon Brion.
+      </Paragraph>
+    </>
+  ),
+  globals: {
+    dir: 'ltr',
+    lang: 'en',
+    title: 'When the Pawn...',
   },
   parameters: {
+    docs: {
+      description: {
+        story: `Een Heading (level 1) met een tekst die met line breaks over meerder regels is verdeeld.
```

Comment:
Volgens mij klopt deze beschrijving niet. Deze is zonder line breaks en mogelijk met word breaks?